design/nfs/nfs-provisioner-controlled-by-operator.md

```diff
@@ -0,0 +1,115 @@
+# NFS Provisioner Controlled by Operator
+
+## Summary
+
+NFS Provisioner is a built in dynamic provisioner for Rook NFS. The functionality works fine but has a issue which the provisioner uses the underlying directory for each provisioned PV when provisioning two or more PV in the same share/export. This issue makes across provisioned PVs in the same share/export has the same data.
+
+This hierarchy is the current behaviour of NFS Provisioner when provisioning two PV in the same share/export.
+
+```console
+export
+├── sample-export
+|   ├── data (from PV-A)
+|   ├── data (from PV-B)
+|   ├── data (from PV-A)
+|   └── data (from PV-A)
+└── another-export
+```
+
+Both PV-A and PV-B uses the `sample-export` directory as their data location.
+
+This proposal is to make Rook NFS Provisioner to create sub-directory for each provisioned PV when provisioning two or more PV in the same share/export. So it will has a hierarchy like.
```

Comment:
```suggestion
This proposal is to make Rook NFS Provisioner create a sub-directory for each provisioned PV when provisioning two or more PV in the same share/export. So it will have a hierarchy like:
```